addons/mute-project/userscript.js

```diff
@@ -6,7 +6,7 @@ export default async function ({ addon, global, console }) {
   icon.loading = "lazy";
   icon.style.display = "none";
   const toggleMute = (e) => {
-    if (e.ctrlKey) {
+    if (e.ctrlKey || e.metaKey) {
```

Comment:
Oh, this diff shouldn't be here. You should have created another branch.

Reply:
oops
might close this PR then